src/components/application_manager/include/application_manager/commands/command_request_impl.h

```diff
@@ -296,19 +320,22 @@ class CommandRequestImpl : public CommandImpl,
       const hmi_apis::FunctionID::eType& function_id);
 
   /**
-   * @brief Add information for the component of response in case of timeout
-   * @param response Response message, which info should be extended
-   */
-  void AddTimeOutComponentInfoToMessage(
-      smart_objects::SmartObjectSPtr& response) const;
-
-  /**
    * @brief Method transforms AppHMIType to string
    * @param enum AppHMIType app_hmi_type contains enum value
    * @return string of AppHMIType
    */
   std::string AppHMITypeToString(
       const mobile_apis::AppHMIType::eType app_hmi_type) const;
+
+  /**
+   * @brief Adds information to result message "info" param in case of GENERIC_ERROR
+            about component which not repsonded in time
```

Comment:
@VVeremjova all fixed in https://github.com/smartdevicelink/sdl_core/pull/1368/commits/6cb301a444f95a33a443f5c0438a182f3b6c9e16